my-app/src/App.css

```diff
@@ -27,7 +27,36 @@
 .App-link {
   color: #61dafb;
 }
-
+header {
+  width: 1000px;
+  height: 300px;
+  background-image: url("./cruce-caraiman.jpg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+.cart-container {
+  margin: 5px;
+  padding: 10px;
+}
+.cart-container button {
+  margin: 0 5px;
+}
+.cart-container span {
+  margin: 5px;
+}
+.mountains-container {
+  margin: 5px;
+  padding: 10px;
+}
+.mountains-container button {
```

Comment:
dupa cum am spus in alt PullRequest, incearca sa le combini cand e cazul:
ex:
.mountains-container button, 
mountains-container span,
.cart-container button {
   margin: 0 5px 
}